lib/phx_diff_web/live/compare_live.ex

```diff
@@ -65,4 +68,16 @@ defmodule PhxDiffWeb.CompareLive do
   defp page_title(%AppSpecification{} = source, %AppSpecification{} = target) do
     "v#{source.phoenix_version} to v#{target.phoenix_version}"
   end
+
+  @github_url "https://github.com/phoenix-diff/phoenix-diff/tree/master/priv/data/sample-app/"
+  defp github_url(%{phx_new_arguments: arguments, phoenix_version: version}) do
+    @github_url <> Path.join(to_string(version), phx_argument_path(arguments))
+  end
+
+  defp phx_argument_path(arguments) do
+    case PhxNewArgListPresets.preset_from_arg_list(arguments) do
+      {:ok, preset} -> PhxNewArgListPresets.PhxNewArgListPreset.path(preset)
+      _ -> "default"
+    end
+  end
```

Comment:
I think I want to move the Github URL generation logic into a  `PhxDiff.fetch_github_sample_app_base_url(app_specification)` function. 

The existing translation from app_spec to path is done here: https://github.com/phoenix-diff/phoenix-diff/blob/d57038524db41380be82cce22221378839a20eeb/lib/phx_diff/diffs/app_repo.ex#L88-L91

You'd probably want to create a public function and use github repo url (read from a config variable) as the base of that path instead of the local app repo path.

Reply:
Sure! i didn't bother too much about it since i anticipated we'd replace the github links with local ones in the near future. But it doesn't cost much to add some config + build the url in app_repo